python3.9/apigw-pytorch/{{cookiecutter.project_name}}/app/requirements.txt

```diff
@@ -1,4 +1,5 @@
 -f https://download.pytorch.org/whl/torch_stable.html
-torch=={{cookiecutter.pytorch_version}}+cpu
-torchvision=={{cookiecutter.torchvision_version}}+cpu
```

Comment:
(This `TODO` would go into the customer's app, so I don't think we should add it)

But also, I don't know if the `+cpu` versions exist anymore.. I think there's a new "trick" to make it work correctly.. and we probably want to do that instead (add `--index-url` with the `cpu` index)

https://stackoverflow.com/a/78215213/574808